src/app/file-manager/warning-modal/warning-modal.component.html

```diff
@@ -0,0 +1,30 @@
+<h2 mat-dialog-title>
+  <i class="fa fa-exclamation-triangle icon-warning" aria-hidden="true"></i>
+  Are you sure you want to
+  <span *ngIf="data == 'exit'"> leave?</span>
```

Comment:
Not sure what the best platform independent way to handle this is, but on the Mac this should be `quit`, since that is what the operation is called in the UI.